extension_helpers/_utils.py

```diff
@@ -162,7 +127,7 @@ def import_file(filename, name=None):
 
     if name is None:
         basename = os.path.splitext(filename)[0]
-        name = '_'.join(os.path.relpath(basename).split(os.sep)[1:])
+        name = '_'.join(os.path.abspath(basename).split(os.sep)[1:])
```

Comment:
:eyes: hmmm, wat.

Reply:
It doesn't really matter I think, this is just generating a module name which isn't really important? On Windows using relpath causes issues on Azure...

Reply:
Cool